### PR TITLE
Allow to get width without stave

### LINKED ISSUE #245: KeySignature allow to get width without assigned Stave
VexMl is crashing when trying to get the width of the element.

### PR DESCRIPTION
**Pull request by rvilarl**
fixes #245

see https://github.com/stringsync/vexml/pull/233

### CODE REVIEW

**Comment by rvilarl** [comment]
@ronyeh thanks! Now setStave is used with a dummy Stave and commented. I prefer the comment because it is legitimate to use format without Stave to figure out the width.

**Comment by ronyeh** [comment]
Is this code already run through eslint? I see some weird lack of spacing.
Between params and also before the brace.

Also I would just save the dummy stave to a local variable so that I don’t
need to call getStave().

**Comment by rvilarl** [comment]
You are right I normally eslint. I will change it this evening.

File: src/keysignature.ts [hunk 1]
```diff
@@ -240,9 +246,16 @@ export class KeySignature extends StaveModifier {
     }
   }
 
-  // Format and position the modifier
+  /**
+   *  Format and position the modifier.
```
Comment (ronyeh) [inline src/keysignature.ts:250]: Sorry I missed this in my previous review. There is an extra space between \* and Format.

**Review by ronyeh — APPROVED** [review]

File: src/keysignature.ts (hunk 1 above)
Comment (rvilarl) [reply src/keysignature.ts]: done

**Pull request merged**